.github/workflows/ci.yaml

```
@@ -70,14 +70,12 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Vendor dependencies
-        run: go mod vendor
-      - uses: bufbuild/buf-setup-action@v1
+      - name: Install linter
+        uses: bufbuild/buf-setup-action@v1
         with:
           version: "1.20.0"
-      - uses: bufbuild/buf-lint-action@v1
-        with:
-          input: api
+      - name: Run linter
+        run: make lint-proto BUF_LINT_ERROR_FORMAT=github-actions
```

Super-nit: I find this slightly more readable, but I know this is personal preference. If you love it the way it is, please leave it. 100% your call.

```suggestion
      - name: Run linter
        env:
          BUF_LINT_ERROR_FORMAT: github-actions
        run: make lint-proto
```

---

Thanks for the suggestion! :)